test_cli.py

```diff
@@ -1415,35 +1374,37 @@ def stress(
     wait(_run_command([cl, 'run', 'ping -c 1 google.com', '--request-network']), 0)
 
 
-# TODO: can't do this test until we can pass in another CodaLab instance.
-@TestModule.register('copy', default=False)
+@TestModule.register('copy')
 def test(ctx):
     """Test copying between instances."""
     source_worksheet = current_worksheet()
 
-    with temp_instance() as remote:
-        remote_worksheet = remote.home
-        _run_command([cl, 'work', remote_worksheet])
+    with remote_instance(ctx.second_instance) as remote:
 
-        def check_agree(command):
+        def compare_output(command):
             check_equals(
-                _run_command(command + ['-w', remote_worksheet]),
                 _run_command(command + ['-w', source_worksheet]),
+                _run_command(command + ['-w', remote_worksheet]),
             )
 
+        remote_worksheet = remote.home
+        print('Source worksheet: %s' % source_worksheet)
+        print('Remote_worksheet: %s' % remote_worksheet)
+
         # Upload to original worksheet, transfer to remote
         _run_command([cl, 'work', source_worksheet])
         uuid = _run_command([cl, 'upload', test_path('')])
         _run_command([cl, 'add', 'bundle', uuid, '--dest-worksheet', remote_worksheet])
-        check_agree([cl, 'info', '-f', 'data_hash,name', uuid])
-        check_agree([cl, 'cat', uuid])
+        compare_output([cl, 'info', '-f', 'data_hash,name', uuid])
+        # TODO: `cl cat` is not working even with the bundle available
+        # compare_output([cl, 'cat', uuid])
```

Comment:
`Copy` test works now but for some reason `cl cat` fails with no errors and I can't figure it out. @percyliang Is it okay if we create a separate, follow-up issue for this?

Failing with:
![fix_cat](https://user-images.githubusercontent.com/16793796/80555321-cdd1f980-8984-11ea-830d-5d0d61604a33.png)

Reply:
Sure.

Reply:
Follow-up issue: https://github.com/codalab/codalab-worksheets/issues/2228